hyperkube/Makefile

```diff
@@ -70,6 +69,7 @@ $(BINARY): $(BINARY).tmp master-multi.json.build master.json.build safe_format_a
 
 $(BINARY).tmp: $(KUBEROOT)/_output/dockerized/bin/linux/amd64/$(BINARY)
 	cp $(KUBEROOT)/_output/dockerized/bin/linux/amd64/$(BINARY) $(BINARY).tmp
+	cp $(KUBEROOT)/_output/dockerized/bin/linux/amd64/kubectl kubectl.tmp
```

Comment:
I assumbe kubectl is build by the same build... so still need a target for that.  however, it should be ok, but still need to watch out for double build (since both targets use the same build command).

$(KUBEROOT)/_output/dockerized/bin/linux/amd64/kubectl:
<tab> (cd $(KUBEROOT)/build; ./run.sh hack/build-go.sh )